MSAL/src/native_auth/controllers/sign_up/MSALNativeAuthSignUpController.swift

```diff
@@ -328,16 +351,29 @@ final class MSALNativeAuthSignUpController: MSALNativeAuthBaseController, MSALNa
                 correlationId: context.correlationId()
             )
             return .init(.error(error: error, newState: newState))
-        case .redirect,
-             .passwordRequired:
-            let error = ResendCodeError()
+        case .redirect:
+            let error = ResendCodeError(correlationId: context.correlationId())
```

Comment:
just as a confirmation, is the correlation not returned from the backend in case of redirect response. Is it right?

Reply:
I checked and yes, it is returned. When we decided to also get the correlationId from the 2xx responses I forgot to check for the `redirect` case, but it is done now. Thanks for pointing it out 👍